### PR TITLE
Adds @jrbourbeau as maintainer

### PR DESCRIPTION
**Pull request by jrbourbeau**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -39,3 +39,4 @@ extra:
   recipe-maintainers:
     - mrocklin
     - pitrou
```
Comment (pitrou) [inline recipe/meta.yaml]: While you are at it, could you remove me as a maintainer?

**Review by pitrou — COMMENTED** [review]
LGTM, but would be nice if you could remove me as well :-)

File: recipe/meta.yaml (hunk 1 above)
Comment (jrbourbeau) [reply recipe/meta.yaml]: No problem, done : )

**Comment by jakirkham** [comment]
@mrocklin, can you please review? 🙂

**Pull request merged**